lldb/include/lldb/Target/RegisterFlags.h

```diff
@@ -9,10 +9,16 @@
 #ifndef LLDB_TARGET_REGISTERFLAGS_H
 #define LLDB_TARGET_REGISTERFLAGS_H
 
-#include "lldb/Utility/Log.h"
+#include <cassert>
+#include <stdint.h>
+#include <string>
+#include <vector>
```

Comment:
Not needed in this header, move to .cpp if needed there?

Reply:
string and vector are needed, presumably because RegisterFlags is a leaf on the build graph so it gets built in relative isolation.

```
In file included from /home/david.spickett/llvm-project/lldb/source/Target/RegisterFlags.cpp:9:
/home/david.spickett/llvm-project/lldb/include/lldb/Target/RegisterFlags.h:84:28: error: no template named 'vector' in namespace 'std'
                const std::vector<Field> &fields);
```

And I can't move them to cpp because we include RegisterFlags.h first.

Reply:
Sorry, I hadn't expanded the full file to see <vector> being used in code that didn't change.